.github/workflows/benchmark-local.yaml

```diff
@@ -0,0 +1,73 @@
+# cli command to run this from repo
+# given a run, get output (link?)
+# input variable to CLI to allow users to specify where to output their artefacts
+
+name: Run performance comparisons
+
+on:
+  workflow_dispatch:
+    inputs:
+      c1:
+        description: First commit hash
+        required: true
+      c2:
+        description: Second commit hash
+        required: true
+
+jobs:
+  build:
+    runs-on: [self-hosted, linux, x64, bench]
+    strategy:
+      fail-fast: true
+      matrix:
+        commit: ["${{ github.event.inputs.c1 }}", "${{ github.event.inputs.c2 }}"]
```

Comment:
Nit: can we make this more descriptive? `base_commit` and `candidate_commit` maybe.

Reply:
Not using `matrix` (since this isn't a performance-comparison tool anymore, but rather a benchmarking tool).